### PR TITLE
Procedural metering mask that follows target

### PR DESCRIPTION
**Pull request by FrancescoC-unity**
Another PR of the exposure saga.

This one adds a new metering mode with a bunch of options to generate a metering mask procedurally:

image

Advanced mode adds a threshold for cutting extreme values: 
image
This also have an option called "center around exposure target" 

If selected and on the camera this field is set: 

image

Then the mask will be centered around  the selected object

targetMask

For QA: 

I strongly suggest to test this with the metering mask debug mode on.

### CODE REVIEW

File: com.unity.render-pipelines.high-definition/Runtime/PostProcessing/PostProcessSystem.cs [hunk 1]
```diff
@@ -839,6 +839,42 @@ internal ComputeBuffer GetHistogramBuffer()
...
+                if (ShaderConfig.s_CameraRelativeRendering != 0)
+                {
+                    targetLocation -= camera.camera.transform.position;
+                }
+                var ndcLoc = camera.mainViewConstants.viewProjMatrix * (targetLocation);
+                ndcLoc.x /= ndcLoc.w;
+                ndcLoc.y /= ndcLoc.w;
+
+                Vector2 targetUV = new Vector2(ndcLoc.x, ndcLoc.y) * 0.5f + new Vector2(0.5f, 0.5f);
+                targetUV.y = 1.0f - targetUV.y;
```
Comment (sebastienlagarde) [inline com.unity.render-pipelines.high-definition/Runtime/PostProcessing/PostProcessSystem.cs:859]: QA: please be sure to test Target on both windows on Metal,  just to be sure it work on all platform

File: com.unity.render-pipelines.high-definition/Runtime/PostProcessing/Shaders/Exposure.compute [hunk 2]
```diff
@@ -12,6 +12,7 @@ TEXTURE2D(_InputTexture);
 
 #define PREPASS_TEX_SIZE 1024.0
 #define PREPASS_TEX_HALF_SIZE 512.0
+#pragma enable_d3d11_debug_symbols
```
Comment (sebastienlagarde) [inline com.unity.render-pipelines.high-definition/Runtime/PostProcessing/Shaders/Exposure.compute:15]: please comment this line

Comment (FrancescoC-unity) [reply com.unity.render-pipelines.high-definition/Runtime/PostProcessing/Shaders/Exposure.compute:15]: Argh yes, didn't see it first time around, doing now.

**Comment by sebastienlagarde** [comment]
QA: be sure to test on Mac as well

**Comment by sebastienlagarde** [comment]
@JMargevics . I merged to be part of the 10.x - preview package but please test and provide feedback, thanks
yamato is green

**Pull request merged**